spec/routing/routes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe 'routes', type: :routing do
```

Comment:
Inclusion of Devise modules (and thus their routes) has always been explicit so I think we're safe.

LGTM.